x-pack/plugins/security_solution/common/api/quickstart_client.gen.ts

```diff
@@ -1405,6 +1411,26 @@ finalize it.
       })
       .catch(catchAxiosErrorFormatAndThrow);
   }
+  /**
+   * Retrieves resources for an existing SIEM rules migration
+   */
+  async getRuleMigrationResources(props: GetRuleMigrationResourcesProps) {
+    this.log.info(`${new Date().toISOString()} Calling API GetRuleMigrationResources`);
+    return this.kbnClient
+      .request<GetRuleMigrationResourcesResponse>({
+        path: replaceParams(
+          '/internal/siem_migrations/rules/{migration_id}/resources',
```

Comment:
can you use the const here https://github.com/elastic/kibana/pull/199370/files#diff-e3310b223fe8037d3b896f8b2d62771a4c943de42f5991780c7faac8fd7959ebR18 for both of these apis?

Reply:
Nope, same as the version, this comes from openAPI schemas. We have to duplicate the paths in the yaml and the constants.ts 🤷

Reply:
👍🏾 . Should've noticed the `gen.ts`